pkgs/by-name/pr/protonmail-bridge-gui/package.nix

```diff
@@ -0,0 +1,134 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libsecret
+, cmake
+, ninja
+, qt6
+, grpc
+, protobuf
+, zlib
+, gtest
+, sentry-native
+, protonmail-bridge
+}:
+
+let
+  pname = "protonmail-bridge-gui";
+  inherit (protonmail-bridge) version src;
+
+  patches = [
+    ./use-nix-googletest.patch
+    ./designator-order-fix.patch
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DBRIDGE_APP_FULL_NAME='Proton Mail Bridge'"
+    "-DBRIDGE_VENDOR='Proton AG'"
+    "-DBRIDGE_REVISION=${src.rev}"
+    "-DBRIDGE_TAG=v${version}"
+    "-DBRIDGE_BUILD_ENV=dev"
+    "-DBRIDGE_APP_VERSION=${version}"
+  ];
+
+in
+stdenv.mkDerivation {
```

Comment:
`finalAttrs` would be better than `let`/`rec` here. See https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes

Reply:
Yes, much cleaner